src/input.service.ts

```diff
@@ -529,21 +529,25 @@ export class InputService {
         const ev = this.focus.createArcEvent(Direction.LEFT);
         this.handleDirection(ev);
         this.onLeft.emit(ev);
+        this.focus.defaultFires(ev, this.scrollSpeed);
       }
       if (gamepad.right(now)) {
         const ev = this.focus.createArcEvent(Direction.RIGHT);
         this.handleDirection(ev);
         this.onRight.emit(ev);
+        this.focus.defaultFires(ev, this.scrollSpeed);
       }
       if (gamepad.down(now)) {
         const ev = this.focus.createArcEvent(Direction.DOWN);
         this.handleDirection(ev);
         this.onDown.emit(ev);
+        this.focus.defaultFires(ev, this.scrollSpeed);
       }
       if (gamepad.up(now)) {
         const ev = this.focus.createArcEvent(Direction.UP);
         this.handleDirection(ev);
         this.onUp.emit(ev);
```

Comment:
Because I want local subscriptions (arc-submit, etc) to run first before global ones. So the order is local, then global, then default. That makes the most sense to me since local subscriptions has a chance to prevent global subscriptions from firing